cardstack/src/components/Text/Text.ts

```diff
@@ -17,6 +17,8 @@ export type TextProps = ShopifyTextProps<Theme> &
   };
 
 const BaseText = createText<Theme>();
+BaseText.defaultProps = BaseText.defaultProps || {};
+BaseText.defaultProps.allowFontScaling = false;
```

Comment:
How about ?
```suggestion
BaseText.defaultProps = {
  allowFontScaling: false,
};
```

This way we don't need to check for undefined and give a default empty object